### ISSUE TITLE
Not all linters seems to ignore .gitignore files and directories

### ISSUE THREAD

**Issue by VictorRos** [oxsecurity/megalinter#2846]
**Describe the bug**

Linters `htmlhint`, `djlint` and `stylelint` (there could be more) trigger errors from `coverage` folder ignored in `.gitignore`

In `.gitignore`

```shell
...
coverage/
...
```

In `.mega-linter.yaml`

```shell
...
IGNORE_GITIGNORED_FILES: true
...
```

**To Reproduce**

A project that uses JEST to run unit tests.

1. Run unit tests (a `coverage` folder will be generated)
2. Run `MegaLinter` on project without removing `coverage` folder

Without JEST

1. Create a `coverage` folder
2. Add HTML and CSS files with errors
4. Run `MegaLinter` on project without removing `coverage` folder

**Expected behavior**

No errors found by MegaLinter's linters on `coverage` folder.

**Screenshots**

I don't want to share screenshots of errors that may reveal company information.

**Additional context**

I'm updating MegaLinter from `7.1.0` to `7.2.1`.
I encountered this problem with both mentioned versions.

**Comment by VictorRos** [comment]
To avoid this problem I'm currently ignoring `coverage` folder using `ADDITIONAL_EXCLUDED_DIRECTORIES` property.

```yaml
# .mega-linter.yaml
...
ADDITIONAL_EXCLUDED_DIRECTORIES:
  - coverage
...
```

If it's a folder that should be ignored anytime, maybe it could be added it to [default MegaLinter excluded directories](https://github.com/oxsecurity/megalinter/blob/cc674427f08ec0e16bbe06e454907ba21ad9e479/megalinter/utils.py#L72C30-L72C30).

What do you think?

**Comment by nvuillam** [comment]
@VictorRos coverage is not a reserved folder like `.git`  , i'm afraid that this would trigger regressions on some projects that would have used such folder name

But your approach using ADDITIONAL_EXCLUDED_DIRECTORIES seems good :)

**Comment by Kurt-von-Laven** [comment]
This sounds like a duplicate of #2955 that was fixed by #2967.

**Comment by nvuillam** [comment]
Thanks @Kurt-von-Laven :)

**Comment by VictorRos** [comment]
Ok I assume it will be available in `7.4.0`?

**Comment by nvuillam** [comment]
@VictorRos exactly :)
